Sources/muter/main.swift

```diff
@@ -1,30 +1,45 @@
 import Darwin
-import Commandant
 import muterCore
+import ArgumentParser
 
 if #available(OSX 10.13, *) {
-    
-    let registry = CommandRegistry<MuterError>()
-    registry
-        .register(InitCommand())
-        .register(RunCommand())
-        .register(HelpCommand(registry: registry))
-        .main(defaultVerb: RunCommand().verb) { (error) in
-            print("""
-                
-            ⚠️ ⚠️ ⚠️ ⚠️ ⚠️  Muter has encountered an error  ⚠️ ⚠️ ⚠️ ⚠️ ⚠️
-            \(error)
-            
 
-            ⚠️ ⚠️ ⚠️ ⚠️ ⚠️  See the Muter error log above this line  ⚠️ ⚠️ ⚠️ ⚠️ ⚠️
-            
-            If you feel like this is a bug, or want help figuring out what could be happening, please open an issue at
-            https://github.com/muter-mutation-testing/muter/issues
-                
-            """)
-            exit(1)
+    struct MuterCommand: ParsableCommand {
+        static var configuration = CommandConfiguration(
+            commandName: "muter",
+            abstract: "🔎 Automated mutation testing for Swift 🕳️",
+            version: "v13",
```

Comment:
I think you can use the version from the global var (it's defined in `version.swift`

Reply:
I'm not at my computer, but I think the issue was that that was in muter core, and this isn't. But we should probably expose that value publicly so other things can use it. We should also maaaybe try to derive it from the tag or something.

Reply:
gotcha! let's see what fits best in this case then 👍

Reply:
I'm still up for making this public in muterCore, if you want, so you can reference it here. Or, if we're using an Xcode project, stick it in an xcconfig or something